package.json

```diff
@@ -56,6 +56,7 @@
     "debug": "^4.3.3",
     "gray-matter": "^4.0.1",
     "ignore": "^7.0.0",
+    "js-yaml": "^4.1.0",
```

Comment:
i _think_ this change should result in a change to our lockfile. mind running `npm i` and committing the changes?